Sync/TabsPayload.swift

```diff
@@ -8,6 +8,10 @@ import Storage
 import XCGLogger
 import SwiftyJSON
 
+// Int64.max / 1000.
+private let MaxSecondsToConvertInt64: Int64 = 9223372036854775;
+private let MaxSecondsToConvertDouble: Double = Double(9223372036854775);
```

Comment:
Actually... nevermind.. for some reason `Double(Int64.max / 1000)` evaluates to `9223372036854776` in Swift playground.